index.html

```diff
@@ -4975,6 +4975,646 @@ <h2 id="AudioWorkerParamDescriptor">
           </section>
         </section>
       </section>
+      <section>
+        <h2 id="AudioWorklet">
+          The <dfn>AudioWorklet</dfn> interface
+        </h2>
+        <p>
+          The <code>AudioWorklet</code> object allows developers to supply
+          Javascript to process audio on the <a>rendering thread</a>,
+          supporting custom AudioNodes. This processing mechanism ensures the
+          synchronous execution of the script code with other built-in
+          <code>AudioNode</code>s in the audio graph.
+        </p>
+        <p>
+          The <code>AudioWorklet</code> object stores the definitions of custom
+          audio processors and <code>AudioWorkletNode</code>s can be
+          constructed based on these definitions. The registered definition in
+          an AudioWorklet is valid for the entire global scope (i.e.
+          <code>window</code>), therefore any <a><code>AudioContext</code></a>
+          instance in the same global scope can access the registered
+          definition to create a custom node.
+        </p>
+        <p>
+          After importing a module script into the
+          <a><code>AudioWorkletGlobalScope</code></a>, an instance of
+          <a><code>AudioWorkletNode</code></a> can be created in the main
+          global scope, based on any of the definitions in the script. At the
+          same time, the user agent automatically creates an instance of
+          <a><code>AudioWorkletProcessor</code></a>. This counterpart
+          represents an audio processing module that runs on the audio
+          <a>rendering thread</a>. These two components can communicate via
+          message passing.
+        </p>
+        <dl title="interface AudioWorklet : Worklet" class="idl">
+          <dt>
+            Promise&lt;void&gt; import(DOMString moduleUrl)
+          </dt>
+          <dd>
+            <p>
+              Imports a module script at <code>moduleURL</code>. The returned
+              promise is resolved when the code is successfully imported and
+              parsed. It is rejected if any uncaught exception occurs during
```

Comment:
I'll try to include something from Worklet spec - but also we might have to consider omit the detail of importing process and simply point the Worklet spec itself.